controllers/productController.test.js

```diff
@@ -1263,4 +1264,139 @@ describe("Product controllers", () => {
       });
     });
   });
+
+  describe("checkInventoryController", () => {
+    let res;
+    const createRes = () => ({
+      status: jest.fn().mockReturnThis(),
+      send: jest.fn().mockReturnThis(),
+      json: jest.fn().mockReturnThis(),
+    });
+
+    beforeEach(() => {
+      res = createRes();
+      jest.clearAllMocks();
+    });
```

Comment:
There's an existing function you can use
```suggestion
    beforeEach(() => {
      res = createMockRes();
      jest.clearAllMocks();
    });
```